spec/acceptance/foreman_service_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper_acceptance'
+
+describe 'configures puma worker count', :order => :defined do
+  context 'initial configuration with 2 puma workers' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        class { 'foreman':
+          foreman_service_puma_workers => 2,
+        }
+        PUPPET
+      end
+    end
+
+    describe service("foreman") do
+      it { is_expected.to be_enabled }
+      it { is_expected.to be_running }
+    end
+
+    describe command('ps aux | grep -v grep | grep -c "puma: cluster worker"') do
+      its(:exit_status) { is_expected.to eq 0 }
+      its('stdout.chomp') { is_expected.to eq('2') }
+    end
+
+    describe process('puma: cluster worker') do
```

Comment:
let's see if that works

Reply:
It does, even if the `be_running` test fails (it checks differently, and there the substring doesn't match)